tst/lambda/discordbot/test_discordbot.py

```diff
@@ -1,41 +1,401 @@
+from copy import deepcopy
```

Comment:
this file is honestly awful but i'm too lazy to fix it right now, unreadable tests are better than no tests.

Reply:
https://github.com/Darklight-03/minecraft-cloudformation-server/issues/12